avm/res/operational-insights/workspace/data-export/main.bicep

```
@@ -53,3 +54,21 @@ output resourceId string = dataExport.id
 
 @description('The name of the resource group the data export was created in.')
 output resourceGroupName string = resourceGroup().name
+
+// =============== //
+//   Definitions   //
+// =============== //
+
+@export()
+type destinationType = {
+  @description('Required. The destination resource ID.')
+  resourceId: string
+  @description('Optional. The destination metadata.')
+  metaData: destinationMetaDataType?
+}
+
+@export()
+type destinationMetaDataType = {
+  @description('Optional. Allows to define an Event Hub name. Not applicable when destination is Storage Account.')
+  eventHubName: string?
+}
```

```suggestion
  metaData: {
    @description('Optional. Allows to define an Event Hub name. Not applicable when destination is Storage Account.')
    eventHubName: string?
  }?
}
```

If only used in one location why 2 types?

---

I merged these two types into one, but somehow I overlooked your commitable suggestion.... 🤷‍♂️